.eslintrc.cjs

```diff
@@ -30,5 +30,11 @@ module.exports = {
     react: {
       version: 'detect',
     },
+    'import/resolver': {
+      typescript: {
+        alwaysTryTypes: true,
+        project: './tsconfig.json',
+      },
+    },
```

Comment:
Was having issues with types and eslint complaining. Not sure why this wasn't needed before and suddenly now. Want to investigate further before merging, I hate having loads of random config I don't understand, especially if it's not needed!